src/main/java/com/example/springboot/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.example.springboot.service.impl;
+
+import com.example.springboot.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final String DELIMITER = ",";
```

Comment:
unfortunately this won't work with our target csv file - because it contains commas inside the review text

let's just plain-read `List<String>` from the source file in this class at the moment
in the next HW we will find the way to parse each string to some DTO using some csv-parsing library

Reply:
Done